tests/Unit/ProtoTests/BasicGrpcOnly/out/src/Client/BasicGrpcOnlyClient.php

```diff
@@ -86,7 +86,7 @@ private static function supportedTransports()
      *
      *     @type string $apiEndpoint
      *           The address of the API remote host. May optionally include the port, formatted
-     *           as "<uri>:<port>". Default 'basicGrpcOnly.example.com:443'.
+     *           as "<uri>:<port>". Default 'basicgrpconly.example.com:443'.
```

Comment:
woops 😦

Reply:
I rebased with main and this should be the correct name isn't it? The proto defines it with lower case. No idea why it changed now.

Reply:
I meant woops because I introduced this in the previous PR by changing it to lowercase before merging. This would have been fine, but I forgot to update the fixtures (as you've done here).

I'd say it's mostly the fault that the tests can pass even when they're accurate, because they aggressively cache.